charts/internal/shoot-control-plane/templates/firewall/firewall.yaml

```diff
@@ -11,11 +11,19 @@ metadata:
   name: firewall
```

Comment:
Would be nice if the values from the chart would be reflected in the chart's default values file.